### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [mediathekview/zapp#252] [General]
Would a Mastodon account for this project be possible?

**Comment by cemrich** [comment]
This is possible. I opened a new issue and will look into it!

**Comment by cemrich** [comment]
Zapp can be found in the fediverse at https://mastodon.technology/@zapp

**Reply by ghost** [reply]
https://floss.social/@zapp

**Reply by pred2k** [reply]
> https://floss.social/@zapp

Could this Account be verified please? https://joinmastodon.org/verification

Everyone could create a zapp Account on any instance and could spread Fake News
